generators/create-react-app/templates/nginx.conf.ejs

```diff
@@ -0,0 +1,7 @@
+location <%= httpPath %> {
+    rewrite <%= httpPath %>(.*) /$1 break;
+
+    set $upstream "http://<%= packageName %>:3000";
+
+    proxy_pass $upstream;
+}
```

Comment:
I'm a bit annoyed that those files look nearly the same for all packages but I'm not sure how this could be factorized properly (and even if it should be since they could diverge at some point).